src/plugins/muc-views/utils.js

```diff
@@ -127,9 +127,24 @@ export function getAutoCompleteListItem (text, input) {
     return element;
 }
 
-export async function getAutoCompleteList () {
-    const models = [...(await api.rooms.get()), ...(await api.contacts.get())];
-    const jids = [...new Set(models.map(o => Strophe.getDomainFromJid(o.get('jid'))))];
+// models array is globalized to introduce caching of the list from https://search.jabber.network/rooms/1.
+const models = [];
```

Comment:
Let's not call this `models`. The word `model` has a very specific meaning in Converse, it refers to instances of the Backbone/Skeletor [Model](https://github.com/conversejs/skeletor/blob/af329a269841d434f1445e35d67a3381a9186b11/src/model.js#L41) class.

search.jabber.network returns the most popular MUCs right? So perhaps call it `popular_mucs`.